lib/ts/error.ts

```diff
@@ -19,4 +19,8 @@ export default class STGeneralError extends Error {
     constructor(message: string) {
         super(message);
     }
+
+    static isGeneralError(err: any): boolean {
```

Comment:
No.. not boolean, but:
```
static isGeneralError(err: any): err is  STGeneralError {
   return err.isSuperTokensGeneralError === true;
}
```

Also name is something better. Cause otherwise it's `STGeneralError.isGeneralError(err)`. How about naming it `isThisError()`?

Reply:
I was not aware of this syntax, done. Also renamed it to `isThisError`.